lambdas/time-epoch-handler/src/time-handler.ts

```diff
@@ -1,29 +1,27 @@
 import { LambdaInterface } from "@aws-lambda-powertools/commons";
 import { TimeEvent } from "./time-event";
-import { toEpochSecondsFromNow } from "./utils/date-time";
+import { milliseconds, toEpochSecondsFromNow } from "./utils/date-time";
 import { LogHelper } from "../../logging/log-helper";
 import { Context } from "aws-lambda";
 
-const logHelper = new LogHelper();
-
 export class TimeHandler implements LambdaInterface {
+  constructor(private readonly logger: LogHelper = new LogHelper()) {}
   public async handler(event: TimeEvent, context: Context) {
-    logHelper.logEntry(context.functionName, event.govJourneyId);
+    this.logger.logEntry(context.functionName, event.govJourneyId);
```

Comment:
Having to pass in the govJourneyId for the purpose of logging is very iunteresting to me. How are we appending the `govJourneyId` to logs that haven't been produced by a lambda?

Reply:
Yeah, I noticed this was missing from https://github.com/govuk-one-login/ipv-cri-check-hmrc-api/pull/269 where it was decided to add id to all lambda's 

The `id` would be in statemachine logs as well 

https://govukverify.atlassian.net/wiki/spaces/OJ/pages/4062904324/Investigate+Linking+state+machine+executions+with+GOV.UK+Journey+IDs